packages/features/tasker/repository.ts

```diff
@@ -134,15 +136,16 @@ export class Task {
   }
 
   static async cleanup() {
-    return db.task.deleteMany({
-      where: {
-        OR: [
-          // Get tasks that have succeeded
-          whereSucceeded,
-          // Get tasks where maxAttemps has been reached
-          whereMaxAttemptsReached,
-        ],
-      },
-    });
+    // TODO: Uncomment this later
+    // return db.task.deleteMany({
+    //   where: {
+    //     OR: [
+    //       // Get tasks that have succeeded
+    //       whereSucceeded,
+    //       // Get tasks where maxAttemps has been reached
+    //       whereMaxAttemptsReached,
+    //     ],
+    //   },
+    // });
```

Comment:
We want to merge this in?

Reply:
yeah we need to disable cleanup tasks for now https://app.campsite.com/cal/posts/htorqjsf6uee#comment-xqg7zlwo1th4